fhir-persistence-schema/README.md

```diff
@@ -85,7 +85,7 @@ The following sections include common values for `OPTIONS`.
 ```
 --prop-file db2.properties
 --schema-name FHIRDATA
---drop-schema
+--drop-schema-fhir
 --drop-admin
 --confirm-drop
 ```
```

Comment:
maybe we should add all three in this example, or mention in a comment below this update that you can use --drop-schema-oauth and jbatch?

Reply:
sure, I can add the comment

Reply:
ended up adding two different subsections; one for dropping just the fhir schema and one for dropping them all